waspc/data/Generator/templates/sdk/wasp/server/crud/_operationTypes.ts

```diff
@@ -79,7 +79,7 @@ export type GetAllQueryResolved = typeof _waspGetAllQuery
 
 {=# crud.operations.Get =}
 {=^ overrides.Get.isDefined =}
-type GetInput = Prisma.{= crud.entityUpper =}WhereUniqueInput
+type GetInput = SuperJSONObject & Prisma.{= crud.entityUpper =}WhereUniqueInput
```

Comment:
The Prisma accepts inputs that are not serializable (e.g. functions), so this intersections makes it select only the kinds of inputs _that are_.
Otherwise it will fail in the following line as Wasp requires only SuperJSON-serializable inputs.